src/resources/lang/de/crud.php

```diff
@@ -125,16 +125,16 @@
     'browse_uploads' => 'Uploads durchsuchen',
     'clear' => 'Löschen',
     'page_link' => 'Link zur Seite',
-    'page_link_placeholder' => 'http://example.com/your-desired-page',
+    'page_link_placeholder' => 'http://example.com/ihre-gewuenschte-seite',
     'internal_link' => 'Interner Link',
-    'internal_link_placeholder' => 'Interner slug. Bsp: \'admin/page\' (ohne Anführungszeichen) für \':url\'',
+    'internal_link_placeholder' => 'Internes Kürzel. Bsp: \'admin/page\' (ohne Anführungszeichen) für \':url\'',
```

Comment:
"Slug" is a completely unknown word in German (outside the developer bubble). "Kürzel" is not the exactly correct translation, but it comes decently close.